packages/backend/src/ee/services/ai/agents/tests/utils/llmAsJudgeForTools.ts

```diff
@@ -0,0 +1,313 @@
+import { AgentToolCallArgsSchema } from '@lightdash/common';
+import { generateObject } from 'ai';
+import { JSONDiff } from 'autoevals';
+import { compact, differenceWith } from 'lodash';
+import { z } from 'zod';
+import { DbAiAgentToolCall } from '../../../../../database/entities/ai';
+import { getOpenaiGptmodel } from '../../../models/openai-gpt';
+import { defaultAgentOptions } from '../../agent';
+
+const dbToolToToolName = (dbToolName: string) => {
+    switch (dbToolName) {
+        case 'findExplores':
+            return 'find_explores';
+        case 'findFields':
+            return 'find_fields';
+        case 'findDashboards':
+            return 'find_dashboards';
+        case 'findCharts':
+            return 'find_charts';
+        case 'generateTableVizConfig':
+            return 'table';
+        case 'generateTimeSeriesVizConfig':
+            return 'time_series_chart';
+        case 'generateBarVizConfig':
+            return 'vertical_bar_chart';
+        default:
+            throw new Error(`Unknown tool name: ${dbToolName}`);
+    }
+};
```

Comment:
Can you reuse this? https://github.com/lightdash/lightdash/blob/57f243cc05e1661f16b6986b7cd3734d31787a9a/packages/common/src/ee/AiAgent/schemas/visualizations/index.ts#L8-L21